### PR TITLE
refactor: :lock: Fixed the title headings

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="UTF-8">
         <meta name="viewpoint" content="width-device-width, initial-scale=1.0">
-        <title>Tec Magic</title>
+        <title>Spell Bind | Home</title>
```
Comment (kaiwalyakoparkar) [inline index.html:6]: Added the home text

File: menu.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Game Menu</title>
+        <title>Spell Bind | Game Menu</title>
```
Comment (kaiwalyakoparkar) [inline menu.html:6]: Added the head title for game

File: storyline/storyline.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html>
  
 <head>
-        <title>Story</title>
+        <title>Spell Bind | Story</title>
```
Comment (kaiwalyakoparkar) [inline storyline/storyline.html:5]: Added text for story

**Review by vinzvinci — APPROVED** [review]
Much better @kaiwalyakoparkar 💯

**Pull request merged**